rskj-core/src/test/java/co/rsk/peg/federation/P2shP2wshErpFederationTest.java

```diff
@@ -164,6 +174,103 @@ private void signTx() {
             }
         }
 
+
+        private static Stream<Arguments> spendFromP2shP2wshErpFedArgsProvider() {
+            BridgeConstants bridgeMainNetConstants = BridgeMainNetConstants.getInstance();
+            BridgeConstants bridgeTestNetConstants = BridgeTestNetConstants.getInstance();
+
+            return Stream.of(
+                Arguments.of(
+                    bridgeMainNetConstants
+                ),
+                Arguments.of(
+                    bridgeTestNetConstants
+                )
+            );
+        }
+
+        @ParameterizedTest()
+        @MethodSource("spendFromP2shP2wshErpFedArgsProvider")
+        void spendFromP2shP2wshErpFed(
```

Comment:
```suggestion
        void spendFromP2shP2wshErpFedEmergencyKeys(
```
wdyt? the other way makes me think we are spending from the fed members

Reply:
done